crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -43,6 +43,22 @@ pub struct SystemId<I = (), O = ()> {
     pub(crate) marker: std::marker::PhantomData<fn(I) -> O>,
 }
 
+impl<I, O> SystemId<I, O> {
+    /// Get the underlying [`Entity`] value of a [`SystemId`].
```

Comment:
I'd like to see a bit of a cautionary note here.

Reply:
What should the user be cautioned against & why would we caution them here? The invariant are checked in `World::run_system_with_input` which is a fallible function.

Reply:
I just restored the original docs.